### DISCUSSION TITLE
How to share scripts with community?

### DISCUSSION THREAD

**Discussion by stevebarakat** [johnlindquist/kit#430] [Q&A] [answered]
Hi. How can I add a script I made to the "Community Scripts"? I've already created a Gist with a shareable link.

**Comment by johnlindquist** [comment, marked as answer]
tl;dr: Use the [Share](https://github.com/johnlindquist/kit/discussions/categories/share) category

## Generating community script pages

![CleanShot 2021-09-13 at 10 22 22](https://user-images.githubusercontent.com/36073/133120871-5f4085c8-de70-4224-8dab-c621331b0246.png)

When you use the "prep for discussion" on a script and post it in the [Share](https://github.com/johnlindquist/kit/discussions/categories/share) category, you will get a page on scriptkit.com with github username like so*:

https://www.scriptkit.com/dealingwith

scriptkit.com queries the github discussions API to pull in the posts.

*This will happen the next time I trigger a build for scriptkit.com, I'll set it to trigger automatically in the future.
** I still need to have the main community page aggregate the generated share pages

**Reply by stevebarakat** [reply]
Okay. I added it. I really just wanted to learn how to do it. I'm not sure I want to keep it. It's just a simple script that I made that announces and lists OS info (primarily for learning how to make scripts). So, hypothetically, if I wanted to delete it in the future, do I just delete the discussion on Github?

**Reply by johnlindquist** [reply]
I triggered a build: Here's what your page looks like: https://www.scriptkit.com/stevebarakat

Yup, deleting the shared discussion would remove it from your page 👍

**Reply by stevebarakat** [reply]
Nice! Thank you!!
